prow/config/trustworthy-jwt.yaml

```diff
@@ -1,6 +1,8 @@
 # This configs KinD to spin up a k8s cluster with trustworthy jwt (Service Account Token Volume Projection) feature.
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  MixedProtocolLBService: true
```

Comment:
I think we need to be careful to ensure we don't try to run these tests on old Kubernetes versions (which run in postusbmit).

will unknown featureGates be silently ignored or fail hard? If they are silent, we can just use RequireMaxVersion in the test. if fail hard, need some more logic in test script I think to pick the config

Reply:
Oh you already did the RequireMaxVersion part

Reply:
Damn! I realised I had issed `!` which resulted in tests skipping when version >= 1.21. Fixed it :D

Reply:
We have a problem. For kind version 1.19 (possibly below that), the feature gate causes issues. So post-submits blow up. Gotta do something on that. :(

Reply:
Ok. I need to dig into `test-infra`. 🤦🏼

Reply:
I have fixed the test infra by turning on `MixedProtocolLBSupport` for K8s >= 1.20 as suggested. Although it is not yet merged (hoping it would be merged), I'll mark this resolved.